doc/man7/migration_guide.pod

```diff
@@ -854,6 +870,14 @@ L<OSSL_SELF_TEST_get_callback(3)> and L<OSSL_SELF_TEST_set_callback(3)>
 
 =item -
 
+L<SSL_load_client_CA_file(3)>
+
+=item -
+
+L<SSL_CTX_new(3)>
```

Comment:
Err...are these in the correct section? This is "New functions that use a Library context". But these are old functions that now have an _ex counterpart.

Reply:
Ahh yeah you are right.. I have trouble with these because the item lists are so large.. Will fix up.